.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,67 +3,62 @@ name: New release template
-- [ ] Run pipeline [Release .Net out-of-proc SDK](https://durabletaskframework.visualstudio.com/Durable%20Task%20Framework%20CI/_build?definitionId=29) to create the new package and publish it to the ADO feed for testing.
+- [ ] Run pipeline [Release .Net out-of-proc SDK](https://azfunc.visualstudio.com/internal/_build?definitionId=657) to create the new package and publish it to the ADO feed for testing.
 
-**Prep Release (assigned to: )**
+**Prep WebJobs and Worker Extensions Release (assigned to: )**
 _Due: <2-business-days-before-release>_
 - [ ] Update DTFx packages and Analyzer versions at WebJobs.Extensions.Durabletask.csproj and check if we need to update the WebJobs.Extensions.Durabletask version.
 - [ ] Locally, run `dotnet list package --vulnerable` to ensure the release is not affected by any vulnerable dependencies.
 - [ ] Review the [Dependabot vulnerability alerts](https://github.com/Azure/azure-functions-durable-extension/security/dependabot) and address them. Note: code samples / test projects _may_ be excluded from this check.
-- [ ] Add the Durable Functions package to the [ADO test feed](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_artifacts/feed/durabletask-test).
 - [ ] Check for package size, make sure it's not surprisingly heavier than a previous release.
 - [ ] Update .NET Isolated SDK version at Worker.Extensions.Durabletask.csproj and check if we need to update the Worker.Extensions.Durabletask version.
-- [ ] Run pipeline [Release .Net Isolated Worker Extension](https://durabletaskframework.visualstudio.com/Durable%20Task%20Framework%20CI/_build?definitionId=30) to create the new package and add it to the ADO feed for testing.
-- [ ] Merge (**choose create a merge commit, NOT squash merge**) dev into main. Person performing validation must approve PR.
-- [ ] Keep branch `v3.x` updated with branch `dev`. Do not merge PRs that are specific to Durable Functions v2.
+- [ ] Run [the extension release pipeline](https://azfunc.visualstudio.com/internal/_build?definitionId=673) to create the new WebJobs.Extensions.Durabletask and Worker.Extensions.Durabletask packages and add them to the [ADO test feed].(https://azfunc.visualstudio.com/internal/_artifacts/feed/durabletask-internal) for testing.
+- [ ] Cherry-pick any PRs that need to be in the `v2.x` branch
 
 **Validation**
 _Due: <1-business-days-before-release>_
 - [ ] Run private performance tests and ensure no regressions. **(assigned to: )**
-- [ ] Smoke test Functions V1, Functions V2, and Functions V3 .NET apps. **(assigned to: )**
+- [ ] Smoke test Functions V2 and Functions V3 .NET apps if you are releasing WebJobs.Extensions.DurableTask v2.x. **(assigned to: )**
```

Comment:
Do we need this to be manual still or do we have coverage with the smoke tests that are run in the CI? Same question for the steps below, which talks about Netherite, MSSQL, and .NET Isolated.

Reply:
They don't need to be manual since the CI smoke tests cover these scenarios. I think in practice, I always do a couple of smoke tests just to double check, so I kept these steps here. Looking at it again, my preference would be to only keep the following smoke test for .NET Isolated to verify that the correct version of the WebJobs extension is loaded.

> Smoke test .NET isolated apps. **(assigned to: )** - check that the correct version of the webjobs extension is loaded by going to bin\Debug\net8.0\.azurefunctions\Microsoft.Azure.WebJobs.Extensions.DurableTask.dll, right click on Properties, go to the Details tab and check the version

I'm open to other suggestions though! @nytian - what do you think?

Reply:
Thanks for the suggestion! I'm fine with only keeping the .NET isolated smoke tests or even remove them all.

I usually ran the .NET isolated smoke tests because, I once forgot to update the assembly version in a Netherite release. However, now that I found that all our repositories are set up with build.target to automatically update the assembly version, that risk has been addressed. So, I think it's safe to remove these smoke tests, or just keep the isolated one.

Reply:
Thanks Naiyuan! I'll just keep the isolated one for now.